resources/model/OpenStudio.idd

```diff
@@ -5200,6 +5200,7 @@ OS:Schedule:File,
        \default No
   N4;  \field Minutes per Item
        \type choice
+       \default 60
```

Comment:
See where default is hardcoded in E+ source code:

https://github.com/NREL/EnergyPlus/blob/93421b9a9d4eb1656e7fa123ded715504d107051/src/EnergyPlus/ScheduleManager.cc#L1752